### PR TITLE
yt-dlp: update to version 2021.12.01

### PR DESCRIPTION
**Pull request by jdambron**
yt-dlp: update to version 2021.12.01

### CODE REVIEW

File: bucket/yt-dlp.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
                 "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp.exe"
             },
             "32bit": {
-                "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp_x86.exe#/yt-dlp.exe"
```
Comment (rashil2000) [inline bucket/yt-dlp.json:27]: Why did you change this?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/yt-dlp.json (hunk 1 above)
Comment (jdambron) [reply bucket/yt-dlp.json:27]: @rashil2000 It seems they changed the URL in the last version. I replaced with the correct one.

Comment (rashil2000) [reply bucket/yt-dlp.json:27]: No, you need to keep the `#/yt-dlp.exe`, it's not a part of the URL, it is for Scoop to rename the file.

**Comment by jdambron** [comment]
@rashil2000  Thank you 👍
